File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Popup.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class Popup{

	@SerializedName("confirmDialogRenderer")
	private ConfirmDialogRenderer confirmDialogRenderer;

	public void setConfirmDialogRenderer(ConfirmDialogRenderer confirmDialogRenderer){
		this.confirmDialogRenderer = confirmDialogRenderer;
	}

	public ConfirmDialogRenderer getConfirmDialogRenderer(){
		return confirmDialogRenderer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Popup.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class Popup : Parcelable, Serializable {
    @SerializedName("confirmDialogRenderer")
    var confirmDialogRenderer: ConfirmDialogRenderer? = null
    override fun toString(): String {
        return "Popup{" +
                "confirmDialogRenderer = '" + confirmDialogRenderer + '\'' +
                "}"
    }
}